### ISSUE TITLE
Classifier layer for pretrained shufflenet v2 model??

### ISSUE THREAD

**Issue by akashlp27** [TropComplique/ShuffleNet-tensorflow#11]
hello,
As mentioned layers for different architecture in this website https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html

is there any know layer for shufflenet v2???
